e2e/topolvm-example.mk

```diff
@@ -0,0 +1,98 @@
+KUBERNETES_VERSION?=1.21.1
+KUSTOMIZE_VERSION=3.8.9
+TOPOLVM_VERSION=0.8.3
+
+KIND_CLUSTER_NAME=autoresizer-e2e
```

Comment:
Would you modify `topolvm/example/Makefile` so that KUBERNETES_VERSION, KIND_CLUSTER_NAME, and TMPDIR can be changed?

Reply:
I made a PR for topolvm/example/Makefile.
https://github.com/topolvm/topolvm/pull/326
Could you check it?

Reply:
Thank you, I fixed Makefile.
https://github.com/topolvm/pvc-autoresizer/pull/44/commits/a72676f71ea34ab13e35334f690b6b31e314c21f